### PR TITLE
Update infra CI/CD prod deployment guide to use latest version

### PR DESCRIPTION
**Pull request by yorinasub17**
This updates the infrastructure CI/CD deployment guide to use the latest version of `module-ci`. Note that this turns off various features used in the app CI/CD workflow.

### CODE REVIEW

**Comment by josh-padnick** [comment]
@yorinasub17 Will hold off on merging until you get another reviewer from the Infra Team (unless you request otherwsie), but see also [this thread on Slack](https://gruntwork-io.slack.com/archives/CGN4WCHGR/p1597898004000600) for an additional update you may want to consider.

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1347]:
```suggestion
  You should limit the actions that can be performed within the ECS tasks due to its powerful IAM permissions. The ECS Deploy
```

I'd suggest a slight rephrasing here as it could be confusing which actions you're referring to - IAM "actions" or "script actions"

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1397]: Soon customers will be able to just use the service-catalog module! Are you expecting to update this again once released?

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1824]:
```suggestion
      DynamoDBLocksTableAccess = {
```

Comment (bwhaley) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1824]: FYI, I've updated this in our current deployment as well.

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1840]:
```suggestion
    infrastructure_live_repositories        = ["git@github.com:<YOUR_ORG>/infrastructure-live.git"]
```

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1871]:
```suggestion
      DynamoDBLocksTableAccess = {
```

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (bwhaley) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1938]:
```suggestion
`--describe-containers` command:
```

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc]: Makes sense! Updated to clarify this is referring to scripts, and also expanded a bit why you want to limit the scripts more: e6300bf7

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1397]: Yes.

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc]: Ah good idea!

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1840]: Woops! Good catch!

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1938]: Good catch!

**Comment by yorinasub17** [comment]
UPDATE: d2a030d6deb2bde6a9f81e36e331dfed9f488e01

I went through the modules to test run until just before circleci config and found a few issues so went through and fixed them.

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [inline _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1138]: @josh-padnick what do you think of this alternative wording?

Comment (josh-padnick) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1138]: Sounds great!

**Comment by yorinasub17** [comment]
UPDATE: rebased on master.

@josh-padnick since I haven't changed anything since https://github.com/gruntwork-io/gruntwork-io.github.io/pull/355#pullrequestreview-474681448 and have @bwhaley 's approval, is it ok to approve => merge => ship this?

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc]: Updated in cc34148d

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1782]: Done: f55bf1d7

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc]: Done: f55bf1d7

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc:1858]: This is a bit difficult to package given that it needs to be pulled in in terragrunt, but yes we should have that.

File: _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc
Comment (yorinasub17) [reply _posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc]: Good point. Updated: 0b6e198c

**Comment by yorinasub17** [comment]
Thanks for the review @brikis98 ! Just finished addressing all your comments.

**Review by brikis98 — APPROVED** [review]
This looks great, thanks Yori!

**Comment by yorinasub17** [comment]
@josh-padnick @eak12913 @oredavids Got the approval from Jim that this is ready to go. Anything else I need to do to get this merged and shipped?

**Comment by josh-padnick** [comment]
Merging now, Yori. In fact, for PRs like this where it's more about the content than the UX or website update, we were just waiting for you and Jim to sort things out. I think letting us know explicitly it's ready for merge is the way to go for now.

**Pull request merged**
